wrotich/twitter-lite-backend
# #165372947 A user should see the time a tweet is submitted

#### What does this PR do?
- Enables a user to see the time at which a tweet was submitted
#### Description of Task to be completed?
- ensure that each tweet should have a time that it was submitted on
- Ensure that if it was less than 24 hours ago, it should give the hourly count
- if it was more than 24 hours ago, it should give the date, e.g. Monday, April 15
#### How should this be manually tested?
- Checkout to this branch.
- setup the database if you haven't by running:
`rake db:create`
`rake db:migrate`
- Start the server `rails s`
- Go to postman and try posting a tweet. You should be able to see the time at which a single tweet was submitted. Also, you can fetch all the previous tweets and you can also see the time at which each of them was submitted.
#### Any background context you want to provide?
- None
#### What are the relevant PT Stories?
[#165372947](https://www.pivotaltracker.com/story/show/165372947)
